package-lock.json

```diff
@@ -3804,35 +3443,30 @@
         },
         "shebang-command": {
           "version": "1.2.0",
-          "resolved": false,
-          "integrity": "sha1-RKrGW2lbAzmJaMOfNj/uXer98eo=",
+          "bundled": true,
```

Comment:
There are a bunch of these changes from "resolved=false" to "bundled=true", what does this effectively mean?

Reply:
Not sure -- I re-made the package-lock.json by deleting the original on (and any installed packages), then running npm install. 

I did a quick search for these, but I'm still not sure of the difference.

Reply:
"bundled=true" makes me suspicious of whether this is going to affect the size of the final MPM or something like that so I think I'd prefer to know what it does before approving the change. @ofrobots do you know what the possible effect here might be?

Reply:
These are showing up because, it seems, that [nyc](https://npm.im/nyc) decided to list a `bundledDependencies` section in its latest release. What this means is that the `nyc` tarball from npm comes bundled together with these dependencies of `nyc`, that the authors have decided, for whatever reason, should come atomically rather than having npm fetch them individually. Think of it as being analogous to static linking.

I am not concerned about this since `nyc` is a `devDependency`. Secondly our `package-lock.json` has no impact to our users – it only affects the module developers (us) and the CI.

Reply:
Link to nyc [package.json](https://github.com/istanbuljs/nyc/blob/bc9ffe59370da43daa62853992402d83c0d17dd4/package.json).